packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -306,6 +321,33 @@ EmbedFlutterFrameworks() {
     RunCommand codesign --force --verbose --sign "${EXPANDED_CODE_SIGN_IDENTITY}" -- "${xcode_frameworks_dir}/App.framework/App"
     RunCommand codesign --force --verbose --sign "${EXPANDED_CODE_SIGN_IDENTITY}" -- "${xcode_frameworks_dir}/Flutter.framework/Flutter"
   fi
+
+  AddObservatoryBonjourService
```

Comment:
How come this works here and not for add-to-app? Because our template build phase puts xcode_backend.sh later than where cocoapods can help us put it?

Reply:
> Because our template build phase puts xcode_backend.sh later than where cocoapods can help us put it?

Bingo.  I put some more details in https://github.com/flutter/flutter/issues/63893#issuecomment-684126922.